components/frontend/src/widgets/DivWithHTML.css

```diff
@@ -0,0 +1,8 @@
+@media print {
+    /* Make sure no scrollbar appears and the HTML content is printed completely */
+    .with-html {
+        height: auto !important;
+        overflow: visible !important;
```

Comment:
Does this never create issues i.e. with some flexbox with absolute sizing?

Reply:
Never say never, but the `DivWithHTML` component is only used in the comment table cell, so I don't expect such issues.